scripts/percona-release.sh

```diff
@@ -120,11 +120,14 @@ function show_help {
   echo "  Example: $(basename ${0}) enable tools release"
   echo "  Example: $(basename ${0}) enable-only ps-80 experimental"
   echo "  Example: $(basename ${0}) setup ps57"
+  echo "  Example: $(basename ${0}) setup -y ps57"
+  echo "  Example: $(basename ${0}) setup ps57 -y"
```

Comment:
I think this is excessively. Let's leave only one example `setup -y`. It will be our recommended way. But if user add `-y` at the end it will also work.

Reply:
Done